demos/controlgroup/splitbutton.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Controlgroup - Split Button</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/button.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/menu.js"></script>
+	<script src="../../ui/position.js"></script>
+	<script src="../../ui/selectmenu.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "select" ).selectmenu({
+			classes: {
+				"ui-selectmenu-button": "ui-button-icon-only"
+			},
+			change: function(){
+				$( ".output" ).append( "<li>" + this.value + "</li>" );
+			}
+		});
+		$( ".controlgroup" ).controlgroup();
```

Comment:
So when you want to customize widgets inside a controlgroup, you init them manually, otherwise auto-init by controlgroup?